test/fixture.h

```diff
@@ -59,6 +59,7 @@ class TestFixture : public ErrorLogger {
     std::string testToRun;
     bool quiet_tests{};
     bool dry_run{};
+    bool mNewTemplate{};
```

Comment:
I think name `mNewTemplate` is unclear. how about two flags `show_column` and `show_id`?

Reply:
This is just a temporary flag until I have converted all tests. It is not supposed to be optional.

Reply:
ok